lighthouse-core/gather/gather-runner.js

```diff
@@ -105,9 +115,10 @@ class GatherRunner {
   /**
    * @param {Driver} driver
    * @param {{requestedUrl: string, settings: LH.Config.Settings}} options
+   * @param {string[]} LighthouseRunWarnings
```

Comment:
you'll need to update from master, `LighthouseRunWarnings` isn't a `string[]` anymore after #10630